packages/shared/src/components/post/PostWidgets.tsx

```diff
@@ -32,6 +34,7 @@ export function PostWidgets({
         className="hidden tablet:flex pt-6"
         contextMenuId="post-widgets-context"
       />
+      {post.type === PostType.Collection && <CollectionsIntro />}
```

Comment:
Is there anyway we could inject this only in the collection modal for instance?

Reply:
We don't have yet the modal/page for rendering collections. We can probably add it if we get a separate implementation of the PostContent for Collections.

Reply:
Cool let's leave it for now, maybe can be a end refactor once we have that :)